recipes/caf/all/conanfile.py

```diff
@@ -0,0 +1,140 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+
+
+class CAFConan(ConanFile):
+    name = "caf"
+    description = "An open source implementation of the Actor Model in C++"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/actor-framework/actor-framework"
+    topics = "conan", "caf", "actor-framework", "actor-model", "pattern-matching", "actors"
+    license = "BSD-3-Clause, BSL-1.0"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "log_level": ["ERROR", "WARNING", "INFO", "DEBUG", "TRACE", "QUIET"],
+        "openssl": [True, False]
+    }
+    default_options = {"shared": False, "fPIC": True, "log_level": "QUIET", "openssl": True}
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+    _cmake = None
+
+    @property
+    def _is_static(self):
+        return 'shared' not in self.options.values.keys() or not self.options.shared
+
+    @property
+    def _has_openssl(self):
+        return 'openssl' in self.options.values.keys() and self.options.openssl
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+            if self.settings.arch == "x86":
+                del self.options.openssl
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("actor-framework-" + self.version, self._source_subfolder)
+
+    def requirements(self):
+        if self._has_openssl:
+            self.requires("openssl/1.1.1j")
+
+    def configure(self):
```

Comment:
```suggestion
    def validate(self):
```

https://blog.conan.io/2020/12/14/New-conan-release-1-32.html